pkg/rpc/agent_redis_request_reply_test.go

```diff
@@ -25,7 +41,7 @@ func TestAgent(t *testing.T) {
 		_ = AgentServe(ctx, c, "agent3", "s3")
 	}()
 	tst := AgentClient{c}
-	time.Sleep(time.Millisecond)
+	time.Sleep(time.Millisecond * 50)
```

Comment:
Please put the `50` to the front, so it is clearer that the sleep will be for 50ms:
```go
time.Sleep(50* time.Millisecond)
```